cluster-operator/src/main/java/io/strimzi/operator/cluster/model/CertUtils.java

```diff
@@ -304,6 +318,21 @@ private static String trustedCertificateVolumeName(CertSecretSource certSecretSo
         return prefix != null ? prefix + '-' + certSecretSource.getSecretName() : certSecretSource.getSecretName();
     }
 
+    /**
+     * Creates the volume mounts for the secret with OAuth trusted TLS certificates
+     *
+     * @param volumeMountList       List where the volume mounts will be added
+     * @param tlsVolumeMountPath    Path where the TLS certs should be mounted
+     * @param secretName            Prefix used for the volume names to distinguish when the same Secret is mounted for
+     *                              different purposes (e.g. different cluster connections in MM2)
+     */
+    public static void createOauthTrustedCertificatesVolumeMounts(List<VolumeMount> volumeMountList, String tlsVolumeMountPath, String secretName) {
```

Comment:
Similarly to above this isn't an oauth specific method, so maybe it should be called `addNewVolumeMount`?